frame/democracy/src/tests.rs

```diff
@@ -274,3 +274,15 @@ fn big_nay(who: u64) -> AccountVote<u64> {
 fn tally(r: ReferendumIndex) -> Tally<u64> {
 	Democracy::referendum_status(r).unwrap().tally
 }
+
+/// note preimage without registering.
+/// randomize the data to make sure the preimage was not noted/registered before.
```

Comment:
randomization is probably not right here. better to have some incrementer which actually ensures the same value never appears twice, otherwise, at one random point, we may get a collision.